pkg/remote/aws/test/iam_role_policy_attachment_multiple/.terraform.lock.hcl

```diff
@@ -0,0 +1,20 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
This file is useless

Reply:
should we add version in the associated tf to lock something or just delete it ?

Reply:
Just delete it